src/emulator/functionsEmulator.ts

```diff
@@ -229,7 +231,16 @@ export class FunctionsEmulator implements EmulatorInstance {
       const projectId = req.params.project_id;
 
       const reqBody = (req as RequestWithRawBody).rawBody;
-      const proto = JSON.parse(reqBody.toString());
+      let proto = JSON.parse(reqBody.toString());
```

Comment:
nit: Consider renaming this (now that it is no longer a proto)

Reply:
This is true, but `proto` name is used throughout the `functionsEmulator.ts` and ultimately used to include in `FunctionsRuntimeBundle`. I'm leaning towards cleaning this up when we start to trim `frb` more aggrssively.